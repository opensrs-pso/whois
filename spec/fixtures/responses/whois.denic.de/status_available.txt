Domain: googlededewdedewdewde.de
Status: free
