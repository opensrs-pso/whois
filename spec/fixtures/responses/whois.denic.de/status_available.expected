#disclaimer
  should: %s == nil


#domain
  should: %s == "googlededewdedewdewde.de"

#domain_id
  should: %s raise_error(Whois::AttributeNotSupported)


#status
  should: %s == :available

#available?
  should: %s == true

#registered?
  should: %s == false


#created_on
  should: %s raise_error(Whois::AttributeNotSupported)

#updated_on
  should: %s == nil

#expires_on
  should: %s raise_error(Whois::AttributeNotSupported)


#registrar
  should: %s == nil

#registrant_contacts
  should: %s CLASS(array)
  should: %s == []

#admin_contacts
  should: %s CLASS(array)
  should: %s == []

#technical_contacts
  should: %s CLASS(array)
  should: %s == []


#nameservers
  should: %s CLASS(array)
  should: %s == []
